main.py

```diff
@@ -53,6 +53,10 @@ def get_uptime_seconds():
     "node_memory_MemAvailable_bytes": "sys.mem.avail",
     "node_memory_MemFree_bytes": "sys.mem.free",
     "node_memory_MemTotal_bytes": "sys.mem.total",
+    # swap
+    "node_memory_SwapCached_bytes": "sys.mem.swap.cached",
```

Comment:
@seanshahkarami do you think this should be

`sys.swap.cached`

Reply:
I think that's a personal preference.

I don't think there's anything strictly depending on _only_ the existing sys.mem.* stuff being there, so this should be fine.

I guess it depends on if you typically want to query and look at all of this at once. If so, I think it's easier to use `sys.mem.swap.*`.